CHANGELOG.md

```diff
@@ -36,6 +38,62 @@
 
 Although every attempt has been made to make this information as accurate as possible, please know there may be things that are omitted, not fully developed yet, or updates since this publication that were not included in the information below. Only the most pressing or significant items have been listed. For the entire list of tickets and or specific information about any given item, please visit the list at [Hyperleder Indy's Jira](https://jira.hyperledger.org/). Once logged in, simply navigate to Projects > Indy.
 
+## 1.7.0 
+### Release date: Apr 26th, 2019
```

Comment:
I think it's better to have the following structure of a Release note:
- Release date
- Component Version Information
- Important Information (rename Additional Information)
- Major Changes (put high-level description of the changes)
- Detailed Changelog (detailed information about tickets)
  - Fixes (sub-section)
  - Changes and Additions (sub-section)

Reply:
`Major Changes` for this release can look as follows:
```
- Audit Ledger 
  - helps keeping all other ledgers in sync
  - helps recovering of pool state by new or restarted nodes
  - can be used for external audit 
- Correct support of multi-signatures
- Configurable Auth Rules in config state
- Stability fixes
```

Reply:
Done.